Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Investigation_Plan_README.md

```diff
@@ -0,0 +1,58 @@
+This playbook handles all the endpoint investigation actions available with Cortex XSIAM, including the following tasks:  
+* Pre-defined MITRE Tactics
```

Comment:
I would change the description because there are no real investigation "actions" here.

I think we should mention that we are using the EDR / endpoint and insights data from multiple log source to detect / find any additional suspicious / malicious behavior  

And the investigation performs a search by:
Pre-defined MITRE Tactics and techniques
Host and attacker activity
..
...

Reply:
This playbook uses the EDR and insights data to detect abnormal and malicious activity and includes the following tasks.
fixed.